flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/legacy/messages/ClusterOverviewWithVersionTest.java

```diff
@@ -31,6 +31,6 @@ protected Class<ClusterOverviewWithVersion> getTestResponseClass() {
 
     @Override
     protected ClusterOverviewWithVersion getTestResponseInstance() {
-        return new ClusterOverviewWithVersion(1, 3, 3, 7, 4, 2, 0, "version", "commit");
+        return new ClusterOverviewWithVersion(2, 6, 3, 1, 3, 7, 4, 2, 0, "version", "commit");
```

Comment:
I think we have to modify because the blocked tms and available tms are expected to be not 0 for the testing, so the totalTaskManager has to be more than 1.